category.php

```diff
@@ -0,0 +1,82 @@
+    "Controversial/CamelCaseParameterName" => "Style",
+    "Controversial/CamelCasePropertyName" => "Style",
+    "Controversial/CamelCaseVariableName" => "Style",
+    "Controversial/Superglobals" => "Security",
+    "CyclomaticComplexity" => "Complexity",
+    "Design/CouplingBetweenObjects" => "Clarity",
+    "Design/DepthOfInheritance" => "Clarity",
+    "Design/EvalExpression" => "Security",
+    "Design/ExitExpression" => "Bug Risk",
+    "Design/GotoStatement" => "Clarity",
+    "Design/LongClass" => "Complexity",
+    "Design/LongMethod" => "Complexity",
+    "Design/LongParameterList" => "Complexity",
+    "Design/NpathComplexity" => "Complexity",
+    "Design/NumberOfChildren" => "Clarity",
+    "Design/TooManyFields" => "Complexity",
+    "Design/TooManyMethods" => "Complexity",
+    "Design/WeightedMethodCount" => "Complexity",
+    "ExcessivePublicCount" => "Complexity",
+    "Naming/BooleanGetMethodName" => "Style",
+    "Naming/ConstantNamingConventions" => "Style",
+    "Naming/ConstructorWithNameAsEnclosingClass" => "Compatability",
+    "Naming/LongVariable" => "Style",
+    "Naming/ShortMethodName" => "Style",
+    "Naming/ShortVariable" => "Style",
+    "UnusedFormalParameter" => "Bug Risk",
+    "UnusedLocalVariable" => "Bug Risk",
+    "UnusedPrivateField" => "Bug Risk",
+    "UnusedPrivateMethod" => "Bug Risk"
+);
+
+Category::$points = array(
+    "CleanCode/BooleanArgumentFlag" => 300000,
+    "CleanCode/ElseExpression" => 200000,
+    "CleanCode/StaticAccess" => 200000,
+    "CodeSize/CyclomaticComplexity" => 500000,
+    "CodeSize/NPathComplexity" => 1000,
+    "CodeSize/ExcessiveMethodLength" => 500000,
+    "CodeSize/ExcessiveClassLength" => 500000,
+    "CodeSize/ExcessiveParameterList" => 25000,
+    "CodeSize/ExcessivePublicCount" => 100000,
+    "CodeSize/TooManyFields" => 100000,
+    "CodeSize/TooManyMethods" => 400000,
+    "CodeSize/ExcessiveClassComplexity" => 500000,
+    "Controversial/Controversial/Superglobals" => 300000,
+    "Controversial/CamelCaseClassName" => 200000,
+    "Controversial/CamelCasePropertyName" => 200000,
+    "Controversial/CamelCaseMethodName" => 200000,
+    "Controversial/CamelCaseParameterName" => 200000,
+    "Controversial/CamelCaseVariableName" => 200000,
+    "Design/ExitExpression" => 200000,
+    "Design/EvalExpression" => 1000000,
+    "Design/GotoStatement" => 900000,
+    "Design/NumberOfChildren" => 2000000,
+    "Design/DepthOfInheritance" => 2000000,
+    "Design/CouplingBetweenObjects" => 700000,
+    "Naming/ShortVariable" => 200000,
+    "Naming/LongVariable" => 100000,
+    "Naming/ShortMethodName" => 400000,
+    "Naming/ConstructorWithNameAsEnclosingClass" => 1000000,
+    "Naming/ConstantNamingConventions" => 800000,
+    "Naming/BooleanGetMethodName" => 500000,
+    "UnusedPrivateField" => 200000,
+    "UnusedLocalVariable" => 200000,
+    "UnusedPrivateMethod" => 200000,
+    "UnusedFormalParameter" => 200000,
```

Comment:
I'm sure there'll be a day when a new check is added to one array and not the other. Is there a way to build this with

```
{
  check => [category, points]
}
```

As the data structure?